go/cmd/dolt/commands/read_tables.go

```diff
@@ -72,7 +72,7 @@ func (cmd ReadTablesCmd) createArgParser() *argparser.ArgParser {
 		{"commit", "Branch or commit hash representing a point in time to retrieve tables from"},
 		{"table", " Optional tables to retrieve.  If omitted, all tables are retrieved."},
 	}
-	ap.SupportsString(dirParamName, "d", "directory", "directory to create and put retrieved table data.")
+	ap.SupportsString(fileParamName, "d", "directory", "directory to create and put retrieved table data.")
```

Comment:
should the flags change to `--file, -f`?

Reply:
This is a good catch, turns out this command was reusing the same const and I broke it by renaming it